integration_tests/mock-names/with-empty-mock-name/__tests__/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+jest.mock('../');
+const importedFn = require('../');
+// empty mock name should result in default 'jest.fn()' output
+const mockFn = jest.fn(importedFn, '');
+
+test('first test', () => {
+  mockFn();
+  expect(mockFn).toHaveBeenCalled();
+  expect(mockFn.mock.calls.length).toBe(1);
```

Comment:
any particular reason why you do this instead of `expect(mockFn)`.toHaveBeenCalledTimes(1)`?

Reply:
No reason. I just copied what was in another test suite in there that I based mine on.